app/src/docker/docker-compose.yml

```diff
@@ -18,7 +18,9 @@ x-mas-vars: &mas-vars
   MAS_COLLECTION_ANNOTS_PATH: ${MAS_COLLECTION_ANNOTS_PATH}
   MAS_COLLECTION_STATUS_PATH: ${MAS_COLLECTION_STATUS_PATH}
   MAS_CREATE_EXAM_ORDER_PATH: ${MAS_CREATE_EXAM_ORDER_PATH}
-  
+  SLACK_EXCEPTION_WEBHOOK: ${SLACK_EXCEPTION_WEBHOOK}
```

Comment:
@advcoder @au2 @rajesh-Krishnasamy do these variables need to be defined anywhere else?